proposals/incubation/kubeflow.md

```diff
@@ -0,0 +1,223 @@
+# Kubeflow CNCF Incubator Project Proposal
+
+## Name of Project:
+
+Kubeflow
+
+## Description
+
+Kubeflow project is an extendable Machine Learning (ML) platform made of multiple distinct components to address specific stages of the ML lifecycle, including:
+
+* **Mode Development** - includes services to create and manage interactive Jupyter notebooks. Users can customize their notebook deployment and their compute resources to suit their data science needs. Supports local workflows with cloud deployment when ready.
```

Comment:
I believe you intended to title this section "Model Development" - with an "L" at the end.